### ISSUE TITLE
Error polling for notifications..

### ISSUE THREAD

**Issue by roughnecks** [GMMan/SteamNotificationsTray#25]
Hello,
not sure when it started happening but if I hover with the mouse over the tray icon, a popup comes out saying:

`Error polling for notifications: Il codice di stato della risp.. `

and nothing more.
I tried logging out and back in, same result.

**Comment by GMMan** [comment]
I think they may have deprecated the old endpoint. Will look into how the new endpoint works.

**Comment by GMMan** [comment]
Might be a bit tricky. Steam currently renders the notifications list directly into its pages, and it doesn't seem to make any API calls until you mark as read. I'll wait for some notifications to gather up again and see if I can capture another call. API of interest: https://steamapi.xpaw.me/#ISteamNotificationService/GetSteamNotifications

**Comment by GMMan** [comment]
Nope, that doesn't work even after supplying `access_token`. Might have to just parse HTML for this, which will suck.

**Comment by roughnecks** [comment]
> Nope, that doesn't work even after supplying `access_token`. Might have to just parse HTML for this, which will suck.

:(

**Comment by GMMan** [comment]
Shower thought: the Steam mobile app should have some sort of regular REST API available to it, unless it's using the same transport as the desktop app?

**Comment by roughnecks** [comment]
Did you find out anything useful?
